# Update doc files regarding JIT support on Solaris.
# Patch was developed in-house and is not appropriate for upstream.

--- pcre2-10.23/doc/pcre2jit.3.orig	2017-03-27 11:01:32.768347866 +0000
+++ pcre2-10.23/doc/pcre2jit.3	2017-03-27 11:02:29.933896903 +0000
@@ -35,6 +35,8 @@
   SPARC 32-bit
 .sp
 If --enable-jit is set on an unsupported platform, compilation fails.
+.sp
+JIT support is not available on OpenIndiana.
 .P
 A program can tell if JIT support is available by calling \fBpcre2_config()\fP
 with the PCRE2_CONFIG_JIT option. The result is 1 when JIT is available, and 0
--- pcre2-10.43/doc/html/pcre2jit.html.orig	2024-02-16 18:04:37.000000000 +0100
+++ pcre2-10.43/doc/html/pcre2jit.html	2024-02-17 14:51:54.200671087 +0100
@@ -64,6 +64,9 @@
 If --enable-jit is set on an unsupported platform, compilation fails.
 </P>
 <P>
+JIT support is not available on OpenIndiana.
+</P>
+<P>
 A client program can tell if JIT support is available by calling
 <b>pcre2_config()</b> with the PCRE2_CONFIG_JIT option. The result is one if
 PCRE2 was built with JIT support, and zero otherwise. However, having the JIT
